### PR TITLE
Update output

### PR DESCRIPTION
**Pull request by kayladnls**
After integration with Zeppelin, a few areas of improvement were needed. This PR addresses those.

### CODE REVIEW

**Comment by localheinz** [comment]
@kayladnls 

Got conflicts!

**Comment by kayladnls** [comment]
Fixing now.

File: src/ResponseBody.php [hunk 1]
```diff
@@ -2,30 +2,26 @@
...
 
 class ResponseBody
 {
     /**
-     * @var \stdClass
-     */
-    private $response;
-
-    /**
      * @var Serializer[]
```
Comment (localheinz) [inline src/ResponseBody.php]: @kayladnls 

Can we update the docblock, too?

```php
/**
 * @var Serializer[]|TopLevelResource[]
 */
```

File: src/ResponseBody.php [hunk 2]
```diff
@@ -2,30 +2,26 @@
...
-    {
-        $this->response = new \stdClass();
-    }
-
     /**
      * @param Serializer $member
+     *
+     * @throws \Exception
      */
     public function addMember(Serializer $member)
```
Comment (localheinz) [inline src/ResponseBody.php]: @kayladnls 

Would it make sense for the `Serializer` interface to extend the `TopLevelResource` interface?

File: src/ResponseBody.php (hunk 1 above)
Comment (kayladnls) [inline src/ResponseBody.php]: Enlighten me on doc blocks. is the `|` meant to indicate "or". In this case it should be both a `Serializer` AND a `TopLevelResource`. Should the docblock be: 

```
/**
 * @var Serializer[] & TopLevelResource[]
 */
```

File: src/ResponseBody.php (hunk 2 above)
Comment (kayladnls) [inline src/ResponseBody.php]: No. Serializers can be Non TopLevelResources.

**Comment by localheinz** [comment]
@kayladnls 

:+1: Looks good to me!

**Pull request merged**